src/Module.js

```diff
@@ -51,9 +51,12 @@ class Module extends Dispatcher {
     this._name = name;
     this.loaded = false
     app.on('stop', this.removeAllListeners.bind(this));
-    this._app.on('load.before', () => {
+    app.on('load.before', () => {
       this.loaded = true
     })
+    this._requestedEvents = {}
+    this._registeredEvents = {}
+    app.after('launch', this._checkMissingEvents.bind(this))
```

Comment:
No, it's totally fine to call an event before it's registered, we don't know it's an error until at least after load, and either way all at once.